src/worldofzuul/Room.java

```diff
@@ -53,17 +69,17 @@ public String getShortDescription() {
     public String getLongDescription() {
         return "You are " + description + ".\n" + getExitString();
     }
-        /**
-         * Gets the exit for the current room
-         * @return the exits from the room 
-         */
-	private String getExitString() {
-        String returnString = "Exits:";
-        Set<String> keys = exits.keySet();
-        for(String exit : keys) {
-            returnString += " " + exit;
-        }
-        return returnString;
+    /**
+     * Gets the exit for the current room
+     * @return the exits from the room 
+     */
+    private String getExitString() {
```

Comment:
Should be spaces instead of tab.

Reply:
fixed in: 718f5ca2b3d2a167093b40a4fc2b4826fc510239